src/hypothesis/strategies.py

```diff
@@ -276,13 +277,12 @@ def floats(
                     allow_nan
                 ))
 
+    min_value = try_convert(float, min_value, 'min_value')
+    max_value = try_convert(float, max_value, 'max_value')
```

Comment:
[This test always passed](https://travis-ci.org/HypothesisWorks/hypothesis-python/jobs/267877445#L2404)